csharp/GSDK_CSharp_Standard/GSDK_CSharp_Standard.csproj

```diff
@@ -9,11 +9,10 @@
     <Version>0.0.0.0</Version>
     <FileVersion>$(Version)</FileVersion>
     <PackageTags>GSDK Playfab Multiplayer Servers</PackageTags>
-    <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
```

Comment:
Just checking to make sure removing the Apache license reference was intentional

Reply:
Yes this property is deprecated. Only seems like there's no replacement since apache 2.0 was already set in the accepted property on line 31 anyway 👍

Reply:
For reference
https://learn.microsoft.com/nuget/create-packages/package-authoring-best-practices#licensing

Reply:
should we add a PackageLicenseExpression then? https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file